_includes/v3.7/release-notes/v3.7.0-release-notes.md

```diff
@@ -0,0 +1,64 @@
+30 Apr 2019
+
+#### Native support for VXLAN encapsulation
+
+Calico now includes native support for encapsulating traffic between
+workloads using VXLAN, in addition to existing support for IP in IP
+encapsulation.
+
+While IP in IP is slightly more efficient in terms of encapsulation
+overhead, there are some networks that do not allow IP in IP traffic,
+for example Azure and DigitalOcean. VXLAN support in Calico means you
+can now use Calico to network workloads in these environments.
+
+ - Felix programs VXLAN routes [felix #1989](https://github.com/projectcalico/felix/pull/1989) (@caseydavenport)
+ - VXLAN encapsulation can now be configured per IP pool [libcalico-go #1065](https://github.com/projectcalico/libcalico-go/pull/1065) (@caseydavenport)
+
+For more information, see the Calico documentation.
+
+#### Optimized denial-of-service protection for host endpoints using XDP
+
+Calico now automatically enforces specific types of blacklist policies
+at the earliest possible point in the packet processing pipeline,
+including offloading to NIC hardware whenever possible.
+
+ - Dataplane: optimize early host-endpoint blacklist rules [felix #1991](https://github.com/projectcalico/felix/pull/1991) (@iaguis)
+ - [calico #2546](https://github.com/projectcalico/calico/pull/2546) (@iaguis)
+
+For more information, see the HOWTO guide and our upcoming blog post.
```

Comment:
Similarly here - could link to https://docs.projectcalico.org/v3.7/security/defend-dos-attack